dns/bind/pkg-descr

```diff
@@ -9,6 +9,10 @@ WWW: https://www.isc.org
 Plugin Changelog
 ================
 
+1.26
```

Comment:
1.26 hasn't been released so you can move these additions to 1.25

Reply:
Thanks have done that.